newsroom/upload.py

```diff
@@ -48,7 +48,7 @@ def get_upload(media_id):
 
     if flask.request.args.get('filename'):
         response.headers['Content-Type'] = media_file.content_type
-        response.headers['Content-Disposition'] = 'attachment; filename="%s"' % flask.request.args['filename']
+        response.headers['Content-Disposition'] = 'attachment; filename=%s' % flask.request.args['filename']
```

Comment:
why removing these? [seems like it should be there](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Disposition)

Reply:
I noticed that `flask.send_file` don't use quotations so that's the reason I removed here. But I'll put it back.